airflow/www/static/js/cluster-activity/live-metrics/DagRuns.tsx

```diff
@@ -0,0 +1,126 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from "react";
+import {
+  Box,
+  BoxProps,
+  Card,
+  CardBody,
+  CardHeader,
+  Center,
+  Flex,
+  Heading,
+  Link,
+  Table,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { useDagRuns } from "src/api";
+import { formatDuration, getDuration } from "src/datetime_utils";
+import LoadingWrapper from "src/components/LoadingWrapper";
+
+const DagRuns = (props: BoxProps) => {
+  const { data, isError } = useDagRuns({
+    dagId: "~",
+    state: ["running"],
+    orderBy: "start_date",
+    limit: 5,
+  });
+
+  return (
+    <Center {...props}>
+      <LoadingWrapper hasData={!!data} isError={isError}>
+        <Card w="100%">
+          <CardHeader textAlign="center" p={3}>
+            <Heading size="md">Dag Runs</Heading>
+          </CardHeader>
+          <CardBody>
+            <Flex flexDirection="column" mb={5}>
+              <Text as="b" color="blue.600">
+                Top 5 longest Dag Runs to finish:
```

Comment:
Same here, maybe move the "5 longest" into the heading a few lines up?

Reply:
Done